lib/kitchen/verifier/pester.rb

```diff
@@ -33,8 +36,20 @@ class Pester < Kitchen::Verifier::Base
 
       default_config :restart_winrm, false
       default_config :test_folder
+      default_config :remove_inbox_packagemanagement, true
+      default_config :remove_inbox_psget, true
+      default_config :remove_inbox_pester, true
       default_config :use_local_pester_module, false
+      default_config :bootstrap, {
+        :repository_url => "https://www.powershellgallery.com/api/v2",
+        :modules => []
+      }
+      default_config :register_repository, []
+      default_config :pester_install, {"Name" => "Pester", "SkipPublisherCheck" => true, "Force" => true, "ErrorAction" => "Stop" }
+      default_config :install_modules, []
       default_config :downloads, ["./PesterTestResults.xml"] => "./testresults"
+      default_config :copy_folders, []
+      default_config :sudo, true
```

Comment:
Yeah this is only to run on non-windows.
On windows  everything runs through winrm, in others it uses SSH as transport.
Because of this, ssh needs to call pwsh, which can be tricky permission-wise without customizing the OS (depending how pwsh has been installed).
so everytime Kitchen-Pester runs a PS command, this will run as `sudo pwsh -f...` if true.